packages/core/.eslintrc.js

```diff
@@ -0,0 +1,123 @@
+module.exports = {
+  env: {
+    jest: true,
+    node: true,
+  },
+  root: true,
+  plugins: ["@typescript-eslint", "import", "prettier"],
+  parser: "@typescript-eslint/parser",
+  parserOptions: {
+    ecmaVersion: 2018,
+    sourceType: "module",
+    tsconfigRootDir: __dirname,
```

Comment:
Eslint needs the relative root to resolve tsconfig. This required the file to be switched to `*.js` rather than `*.json`.